scripts/blender_load_replay.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+r"""
+Script for loading a GFX replay file into Blender. This allows playing back a trajectory from Habitat in Blender.
+
+See the following tutorial for more information on GFX: https://colab.research.google.com/github/facebookresearch/habitat-sim/blob/main/examples/tutorials/colabs/replay_tutorial.ipynb
```

Comment:
Rewrite this to `more information on gfx-replay`. `GFX` is a generic abbreviation for graphics and is too vague.